conda/recipes/meta.yaml

```
@@ -0,0 +1,46 @@
+package:
+  name: nvjitlink
+  version: 0.1.0
+
+source:
+  #url: https://github.com/rapidsai/nvjitlink/archive/refs/tags/0.1.0.tar.gz
+  path: ../../
+
+build:
+  noarch: python
+  script: pip install .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda-nvcc') }}
```

Add a `conda_build_config.yaml` with these contents:
```yaml
c_compiler_version:
  - 11

cxx_compiler_version:
  - 11

cuda_compiler:
  - cuda-nvcc
```

And apply this suggestion:
```suggestion
    - {{ compiler('cuda') }}
```

---

Out of curiosity, is there any reason to prefer this other than "This is what we've done everywhere else"? AFAIK the two should be equivalent (not including the version bits of course).

---

The `compiler('cuda')` syntax replaces this with the value from `cuda_compiler` (`cuda-nvcc` in Bradley's code), which in turn gets suffixed with the `target_platform` (becoming `cuda-nvcc_linux-64` for example), which is then the package name Conda uses to install

---

Right, I understand how it works but my question is whether there's any benefit to specifying `compiler('cuda')` and having the `cuda_compiler` in the CBC vs just using `compiler('cuda-nvcc')`. I know that's what we've done in our other recipes, and I'm fine with following that pattern for consistency, but I wanted to confirm that this was a stylistic suggestion and not functionally different.